src/main/jbake/content/blog/2017/03/14/hawkular-alerting-tutorial.adoc

```diff
@@ -0,0 +1,16 @@
+= Hawkular Alerting: Tutorial Now Available!
+Jay Shaughnessy
+2017-03-14
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, alerting
+
+Hawkular Alerting is a pluggable, API-driven component for adding alerting to your application or tool.  It's one of the many http://hawkular.org[Hawkular] offerings for monitoring and management.  To help quick-start new users, and as a reference for current users, we've released the first five lessons of a new tutorial, with more to follow.  It's a great way to hit the ground running.
+
+To get started just clone or download http://github.com/hawkular/hawkular-alerts[Hawkular Alerting from GitHub].  You'll find the tutorial under the `examples` directory. Start with the README.adoc.  If you don't have an asciidoc extension in your browser you can see the rendered README https://github.com/hawkular/hawkular-alerts/blob/master/examples/README.adoc[directly from GitHub].
```

Comment:
Even if it is trivial, I like the style of copy/paste for ultra lazies.
So, some box code text with the commands ready to start people invites to test.
(In previous blogs is more or less how everyone suggests).